bot/resources/tags/traceback.md

```diff
@@ -1,18 +1,18 @@
-Please provide a full traceback to your exception in order for us to identify your issue.
+Please provide the full traceback for your exception in order to help us identify your issue.
 
 A full traceback could look like:
 ```py
 Traceback (most recent call last):
     File "tiny", line 3, in
         do_something()
     File "tiny", line 2, in do_something
-        a = 6 / 0
+        a = 6 / b
 ZeroDivisionError: integer division or modulo by zero
 ```
 The best way to read your traceback is bottom to top.
 
-• Identify the exception raised (e.g. ZeroDivisionError)  
-• Make note of the line number, and navigate there in your program.  
-• Try to understand why the error occurred.  
+• Identify the exception raised (e.g. `ZeroDivisionError`)  
+• Make note of the line number (in this case 2), and navigate there in your program.  
+• Try to understand why the error occurred (in this case because `b` must be `0`).
```

Comment:
```suggestion
• Try to understand why the error occurred (in this case probably because `b` is `0`).
```

Reply:
Even "most likely" isn't correct. We know categorically that `b` *is* `0` from this error. There's no other way for the error to be raised.

I agree that `is` flows better than `must be` though so will change that.

Also going to update the error message since apparently I forgot to do that.

Reply:
Yup sounds good